DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,374 @@
+ - [Draino](https://github.com/planetlabs/draino) and [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+
+In this tutorial the main focus will be the [Digital Mobius](https://github.com/Qovery/digital-mobius) project. A simplified diagram of the internal flow (logic) is provided as well. Finally, we will have it deployed to a running `DOKS` cluster and observe how well it handles cluster nodes recycling.
+
+Overview and features:
+ - Open source (written in [GO](https://golang.org))
+ - Can be used as a `Kubernetes Deployment` or as a standalone binary (CLI)
+ - [Helm chart](https://github.com/Qovery/digital-mobius/tree/main/charts/Digital-Mobius) source code is available for easy K8S deployment (or on [artifacthub.io](https://artifacthub.io/packages/helm/digital-mobius/digital-mobius))
+
+## DOKS Cluster Setup
+
+**Note:**
+
+Steps **1** to **5** can be skipped if you already have a cluster running in place and the `kubectl context` configured.
+
+1. A `DigitalOcean` [account](https://cloud.digitalocean.com) is required to create the access keys and provision the `DOKS` cluster.
+2. An `API access token` for managing the `DOKS` cluster has to be created next. From your `DigitalOcean` account, go to the [API](https://cloud.digitalocean.com/account/api) section to generate the token:
+
+    ![Applications & API Key](content/img/pt_gen_1st.png)
+
+    Give it a name and ensure that the `Write` scope is checked:
+
+    ![Applications & API Key](content/img/pt_gen_2nd.png)
+
+    In the end, we should have something like this:
+
+    ![Applications & API Key](content/img/api_access_key.png)
+
+    Copy the token value and save it in a local environment variable as we will need it later on (make sure to replace the `<>` placeholder):
+
+    ```bash
+    export DIGITAL_OCEAN_TOKEN="<your_do_personal_access_token>"
+    ```
+3. In order to interact with the cluster from the CLI `doctl` has to be installed. For example, use the following commands on MacOS:
+    
+    ```bash
+    brew info doctl
+    brew install doctl
+    ```
+4. If `doctl` authentication hasn't been set up yet please do it now via:
+   
+    ```bash
+    doctl auth init --access-token "$DIGITAL_OCEAN_TOKEN"
+    ```
+    The output looks similar to the following (token value is masked):
+    ```
+    Using token [*****************************************]
+
+    Validating token... OK
+    ```
+5. Spin up the cluster and wait for it to be provisioned:
+
+    ```bash
+    export CLUSTER_NAME="mobius-testing-cluster"  # change as required
+    export CLUSTER_REGION="lon1"                  # grab a region that's more close to you from: doctl k8s options regions
+    export CLUSTER_NODE_SIZE="s-2vcpu-4gb"        # change as required
+    export CLUSTER_NODE_COUNT=2                   # need 2 nodes at least to test the scenarios
+    export CLUSTER_NODE_POOL_NAME="mbt-np"        # change as required
```

Comment:
What are the "change as required" labels needed for?

If they are not really required, then I'd recommend to drop them to avoid confusion.

Reply:
Thanks for the suggestion. Fixed in the latest commit.